src/libraries/System.Private.CoreLib/src/System/Char.cs

```diff
@@ -591,7 +591,7 @@ public static bool IsControl(string s, int index)
             {
                 ThrowHelper.ThrowArgumentNullException(ExceptionArgument.s);
             }
-            if (((uint)index) >= ((uint)s.Length))
+            if ((uint)index >= (uint)s.Length)
```

Comment:
These ones seem to be touching code "stylistically", we probably should exclude it.

Reply:
W/o the superfluos paranthesis it's easier to read, thus I changed it.